ansible/roles-infra/infra-osp-create-inventory/tasks/main.yml

```diff
@@ -35,18 +35,13 @@
       name: "{{ server | json_query(_name_selector) | default(server.name) }}"
       original_name: "{{ server.name }}"
       groups:
-      #TODO: remove thos tag_*
-      # - "tag_Project_{{stack_tag}}"
-      # - "tag_{{ stack_tag}} | default('unknowns') }}"
       - "{{ server.metadata.ostype | default('unknowns') }}"
+      - "{{ server.metadata.AnsibleGroup | default('') }}"
```

Comment:
Why do you add this line ?

it is done later in the code

Reply:
Whoops, I was testing adding that in the same task and forgot to remove it. I still can't understand why this doesn't work but I don't have time to fix it.